addons/mail/static/src/owl/components/activity/activity.scss

```diff
@@ -0,0 +1,174 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_Activity {
+    display: flex;
+    flex: 0 0 auto;
+    padding: 5px;
+}
+
+.o_Activity_detailsUserAvatar {
+    margin-right: 5px;
+    object-fit: cover;
+    height: 18px;
+    width: 18px;
+}
+
+.o_Activity_icon {
+    position: absolute;
+    top: auto;
+    left: auto;
+    bottom: -5px;
+    right: -5px;
+    width: 25px;
+    height: 25px;
+    object-fit: cover;
+    padding: 6px 5px;
+    text-align: center;
+    line-height: 1.2;
+    border-width: 2px;
+}
+
+.o_Activity_sidebar {
+    display: flex;
+    flex: 0 0 36px;
+    margin-right: 10px;
+    justify-content: center;
+}
+
+.o_Activity_toolButton {
+    padding-top: 0;
+}
+
+.o_Activity_tools {
+    display: flex;
+}
+
+.o_Activity_user {
+    height: 36px;
+    margin-left: 3px;
+    margin-right: 3px;
+    position: relative;
+    width: 36px;
+}
+
+.o_Activity_userAvatar {
+    height: 100%;
+    width: 100%;
+}
+
+.o_Activity_userName {
+    margin-left: 5px;
+}
+
+// ------------------------------------------------------------------
+// Style
+// ------------------------------------------------------------------
+
+$activityDefaultColor: gray('300');
+$activityOverdueColor: darken(theme-color('danger'), 10%);
+$activityPlannedColor: darken(theme-color('success'), 10%);
+$activityTodayColor: darken(theme-color('warning'), 10%);
```

Comment:
add `!default` at the end of each to allow customization

Probably need to prefix them with `o-mail-`, and scss variables use the `-` separating words syntax